docker-compose.yaml

```diff
@@ -56,14 +58,16 @@ services:
       # - S3_USE_PATH_STYLE_ENDPOINT=true
 
       # To customize upload limits
-      # - ANON_MAX_EXPIRE_SECONDS=604800
-      # - MAX_EXPIRE_SECONDS=604800
-      # - ANON_MAX_DOWNLOADS=20
-      # - MAX_DOWNLOADS=20
-      # - ANON_MAX_FILE_SIZE=2684354560
+      # - EXPIRE_TIMES_SECONDS=3600,86400,604800,2592000,31536000
+      # - DEFAULT_EXPIRE_SECONDS=3600
+      # - MAX_EXPIRE_SECONDS=31536000
+      # - DOWNLOAD_COUNTS=1,2,5,10,15,25,50,100,1000
+      # - MAX_DOWNLOADS=1000
       # - MAX_FILE_SIZE=2684354560
+  
   redis:
     image: 'redis:alpine'
+    command: redis-server --appendonly yes
```

Comment:
Do you have any specific reason to set `--appendonly`?

I don't think this is needed. In fact, it is probably a bad idea as it would allow you to recover expired files.

Reply:
Just noticed this point:

> Fix redis container mounting data volume but not saving any data to disk by adding --appendonly yes option

I'm not seeing this issue when running this configuration myself. Could there be a permission problem instead?

Reply:
Ah I hadn't considered the side effect of being able to recover expired files.

I'm used to enabling aof mode for better integrity compared to RDB mode alone, but I think this is one case where plain RDB is fine. I'll remove it.

Reply:
I understand! Yeah, this seems to be the exception.